pkg/model/trace.go

```diff
@@ -15,18 +15,18 @@ import (
 	v1 "github.com/grafana/tempo/pkg/tempopb/trace/v1"
 )
 
-func CombineTraces(objA []byte, objB []byte) (_ []byte, wasCombined bool, _ error) {
+// todo(jpe):
+// - add cross data encoding tests
+
+func CombineTraceBytes(objA []byte, objB []byte, dataEncodingA string, dataEncodingB string) (_ []byte, wasCombined bool, _ error) {
```

Comment:
Its unclear what encoding this function returns the combined bytes in.. can we add a comment before this?

Reply:
good catch, we were always marshalling using the original encoding. This has been updated to support multiple encodings. data is always returned in dataEncodingA.